Leanplum-SDK/Classes/MessageTemplates/LPBaseInterstitialMessageTemplate.m

```diff
@@ -147,6 +164,24 @@ - (void)showPopup
                                              selector:@selector(orientationDidChange:)
                                                  name:UIDeviceOrientationDidChangeNotification
                                                object:nil];
+    
+    [[NSNotificationCenter defaultCenter] addObserver:self
+                                             selector:@selector(appDidBecomeActive:)
+                                                 name:UIApplicationDidBecomeActiveNotification
+                                               object:nil];
+}
+
+- (void)setFocusForAccessibilityElement:(UIView *)accessibleView {
+    UIAccessibilityPostNotification(UIAccessibilityScreenChangedNotification,
+    accessibleView);
+}
+
+- (void)appDidBecomeActive:(NSNotification *)notification {
+    if (self.popupGroup != nil) {
```

Comment:
Nice. Is this how we allow the Alert dialogs to take focus?

Reply:
In this method actually I'm getting the last view and setting the focus with `setFocusForAccessibilityElement` method. This check `if (self.popupGroup != nil)` is not really necessary now, because I'm removing the observer when the popup is closed/removed from the view. So, I'll refactor this.